### ISSUE TITLE
[Support] Resolution issues with Intouch

### ISSUE THREAD

**Issue by michaela-int** [johnboiles/obs-mac-virtualcam#199]
**Describe the issue**
When using the virtual camera in Intouch (my company's videoconferencing platform), screen text is unreadable. If I were to screenshare, this text is readable. 

**To Reproduce**
1.  Start the virtual camera
2. Go to intouch.ideanovatech.com
3. Click on 'Go' to start a meeting
4. Choose the virtual camera as the video input device
5. Connect to Intouch from another browser or ask a friend to join
5. The second connection will be able to see the distorted images. 

**Behavior**
Other meeting attendees cannot read powerpoint slides and they frequently see visible lag when I change slides.  

**Screenshots**
<img width="1675" alt="Screen Shot 2020-08-21 at 3 20 13 PM" src="https://user-images.githubusercontent.com/68753123/90939825-637d1300-e3d2-11ea-91bf-8959c1ac0df1.png">
(I can provide more, but they're basically all the same)

**Desktop (please complete the following information):**
 - OS version: macOS Catalina 10.15.3
 - Application (if applicable): Intouch videoconferencing run on Chrome or Firefox
 - Virtualcam version: 1.2.0
 - OBS version: e.g. 25.0.8

**Additional context**
I have manually set the frame rate to 30fps per #62 and I have set the resolution to 1920x1080 per #118 and #135. Changing the frame rate did help, but the text is frequently still garbled. My color space is 601 and the color range is partial. Intouch is jitsi-based if that helps.

**Comment by ubidefeo** [comment]
@michaela-int 
I ended up on this issue while researching how to fix this for Zoom (which everyone will agree is evil disguised as the end of all problems).
I have tried all the suggestions (and actually have a Zoom Pro account) but nothing worked.
What I do now is to use a cheap 1920x1080p external display as the fullscreen preview of OBS and share that screen as my desktop. I also remove my video input.
I guess some apps apply trickery to webcams but not to screen sharing, and that's what gets you such awful quality
